lib/pages/chat/events/message/message_content_with_timestamp_builder.dart

```diff
@@ -138,8 +138,7 @@ class MessageContentWithTimestampBuilder extends StatelessWidget {
                               Column(
                             crossAxisAlignment: CrossAxisAlignment.start,
                             children: [
-                              event.hideDisplayName(nextEvent) ||
-                                      event.hideDisplayNameInBubbleChat
+                              event.hideDisplayName(nextEvent)
```

Comment:
base on telegram, IMO we have the same logic between direct chat and group chat.